internal/uvm/vpmem.go

```diff
@@ -19,29 +18,44 @@ func (uvm *UtilityVM) allocateVPMEM(ctx context.Context, hostPath string) (uint3
 		if vi.hostPath == "" {
 			vi.hostPath = hostPath
 			log.G(ctx).WithFields(logrus.Fields{
-				logfields.UVMID: uvm.id,
-				"host-path":     vi.hostPath,
-				"uvm-path":      vi.uvmPath,
-				"refCount":      vi.refCount,
-				"deviceNumber":  uint32(index),
-			}).Debug("uvm::allocateVPMEM")
+				"hostPath":     hostPath,
+				"uvmPath":      vi.uvmPath,
+				"refCount":     vi.refCount,
+				"deviceNumber": index,
+			}).Debug("allocated VPMEM location")
 			return uint32(index), nil
 		}
 	}
 	return 0, fmt.Errorf("no free VPMEM locations")
 }
 
+func (uvm *UtilityVM) deallocateVPMEM(ctx context.Context, deviceNumber uint32) error {
```

Comment:
This function doesn't look like it's used in this PR. Is this supposed to be in here?

Reply:
And this is why we need linting.